inc/osvr/Client/LocateServer.h

```diff
@@ -0,0 +1,70 @@
+/** @file
+@brief Header
+
+@date 2016
+
+@author
+Sensics, Inc.
+<http://sensics.com/osvr>
+*/
+
+// Copyright 2016 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_LocateServer_h_GUID_7D10F12E_2A83_48BB_7581_ED7A7F5188AC
+#define INCLUDED_LocateServer_h_GUID_7D10F12E_2A83_48BB_7581_ED7A7F5188AC
+
+
+// Internal Includes
+#include <osvr/Common/GetEnvironmentVariable.h>
+#include <osvr/Util/PlatformConfig.h>
+#include <osvr/Util/APIBaseC.h>
+
+// Library/third-party includes
+// - none
+
+// Standard includes
+#include <string>
+
+namespace osvr {
+    namespace client {
+
+        /** @brief INTERNAL ONLY - get the current server directory, if available. */
+        OSVR_INLINE boost::optional<std::string> getServerBinaryDirectoryPath() {
+            auto server = osvr::common::getEnvironmentVariable("OSVR_SERVER");
+            return server;
+        }
+
+        /** @brief INTERNAL ONLY - get the path to the server executable, if available. */
+        OSVR_INLINE boost::optional<std::string> getServerBinaryPath() {
+            auto binPath = getServerBinaryDirectoryPath();
+            if (binPath.is_initialized()) {
+#if defined(OSVR_WINDOWS)
+                return *binPath + "\\osvr_server.exe";
```

Comment:
If you change above to be `OSVR_SERVER_ROOT` you'd need to add a bin in here, probably from a configured file. Also, not positive how critical the different path separators are, but it's a nice thought :smile: